presets/4.3/other/ELRS_Passthrough.txt

```diff
@@ -0,0 +1,14 @@
+#$ TITLE: ExpressLRS Passthrough Betaflight Settings
+#$ FIRMWARE_VERSION: 3.5
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ STATUS: OFFICIAL
+#$ CATEGORY: OTHER
+#$ KEYWORDS: ELRS Betaflight Passthrough Configure
+#$ AUTHOR: Daniel Appel / Tehllama
+#$ DESCRIPTION: This is just a simplified way of ensuring that Betaflight is correctly configured to enable ExpressLRS to perform receiver flashing via Passthrough
```

Comment:
I am not familliar with ERLS at all, so i could be wrong here:
any words of how a person would revert these settings after he is done with passthrough? Or a person does not need to revert them and can keep flying after he applies them?

Reply:
The reversion is realistically the problem... and why this is potentially a useless preset.
This would allow a user to perform the Betaflight Passthrough method of reflashing an ExpressLRS receiver... but does nothing to restore the previous setting.